components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -442,6 +445,42 @@ export function CreateWorkspacePage() {
     );
 }
 
+function LocalSSHWarning(props: { className: string }) {
+    const isLocalSSHProxyEnabled = useFeatureFlag("gitpod_desktop_use_local_ssh_proxy");
+
+    if (!isLocalSSHProxyEnabled) {
+        return null;
+    }
+
+    return (
+        <div className={props.className}>
+            <Alert light className="bg-gray-100 dark:bg-gray-800" type="info">
+                To make VS Code Desktop open seamlessly we will add a single entry to your local SSH configuration.
+                Gitpod does not read any of your existing SSH configurations.&nbsp;
+                <a
+                    className="gp-link"
+                    href="https://www.gitpod.io/docs/references/ides-and-editors/vscode#ssh-configuration-modifications"
+                    target="_blank"
+                    rel="noreferrer"
+                >
+                    Learn more
+                </a>
+                <div className="flex mt-2 text-sm">
+                    Comment on&nbsp;
+                    <a
+                        href="https://github.com/gitpod-io/gitpod/issues/18109"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        className="gp-link"
+                    >
+                        feedback issue
+                    </a>
+                </div>
```

Comment:
**suggestion**: Thought there was consensus to drop the duplicate feedback link in https://github.com/gitpod-io/gitpod/pull/18112#discussion_r1252178641, but if we think it's valuable to keep, let's move it next to the learn more help link.

```suggestion
                </a>
                &nbsp;&middot;&nbsp;
                <a
                    className="gp-link"
                    href="https://github.com/gitpod-io/gitpod/issues/18109"
                    target="_blank"
                    rel="noreferrer"
                >
                    Send feedback
                </a>
```